client/app/_layout.tsx

```diff
@@ -21,7 +21,7 @@ const RootLayoutNav = () => (
         <Provider store={store}>
           <AuthListener>
             <StatusBar />
```

Comment:
Not sure why I don't see it on the other screens, but maybe should consider making this style light

```suggestion
            <StatusBar style="light" />
```

Reply:
Changed!